Didi/entry address comments (#1007)
telerik/maui-docs: controls/datapager/display-mode.md

The DataPager lets you control the visibility of its visual elements. In addition, you can use Adaptive Display modes to arrange the pager's elements.

> To define which elements can be displayed in the DataPager, use the `DisplayMode` (`enum` of type `Telerik.Maui.Controls.DataPager.DataPagerDisplayMode`) property. The available options arre:

Reviewer: ```suggestion
To define which elements can be displayed in the DataPager, use the `DisplayMode` (`enum` of type `Telerik.Maui.Controls.DataPager.DataPagerDisplayMode`) property. The available options are:
```

The DataPager lets you customize the default arrangement of the elements in the pager by providing the `AdaptiveDisplayModes` property:

> * `AdaptiveDisplayModes` (`IList<DataPagerDisplayMode>`)&mdash;Specifies a list of the desired combinations of elements that should be displayed. The actual elements that will be displayed are the result of the DataPager.`DisplayMode` property, and the size of the control.

Reviewer: ```suggestion
* `AdaptiveDisplayModes` (`IList<DataPagerDisplayMode>`)&mdash;Specifies a list of the desired combinations of elements that should be displayed. The actual elements that will be displayed are the result of the DataPager's `DisplayMode` property and the size of the control.
```

> For example, if you set `AdaptiveDisplayModes` and `DisplayMode` to the following values:

Reviewer: ```suggestion
For example, if you set `AdaptiveDisplayModes` and `DisplayMode` to the values in the snippet below, the DataPager will arrange the elements based on the values set in the `AdaptiveDisplayModes` collection and will skip the elements that are not defined in the `DisplayMode` property&mdash;the `NextPageButton` element.
```

```XAML
<telerik:RadDataPager Source="{Binding Data}"
                      DisplayMode="FirstPageButton,LastPageButton,NavigationComboBox">
    <telerik:RadDataPager.AdaptiveDisplayModes>
        <x:Array Type="{x:Type telerik:DataPagerDisplayMode}">
            <telerik:DataPagerDisplayMode>FirstPageButton, NavigationComboBox, NextPageButton, LastPageButton</telerik:DataPagerDisplayMode>
        </x:Array>
    </telerik:RadDataPager.AdaptiveDisplayModes>
</telerik:RadDataPager>
```

> The DataPager will arrange the elements based on the values set in the `AdaptiveDisplayModes` collection and will skip the elements that are not defined in the `DisplayMode` property&mdash;the `NextPageButton` element.

Reviewer: ```suggestion
```
